test/TestAppUtils/APITestBase.cs

```diff
@@ -17,9 +16,9 @@
 using Microsoft.VisualStudio.TestTools.UnitTesting.Logging;
 #endif
 
-namespace Windows.UI.Xaml.Tests.MUXControls.ApiTests.RepeaterTests
+namespace MUXControlsTestApp.Utilities
 {
-    public class TestsBase
+    public class ApiTestBase
```

Comment:
It doesn't let me comment on the line for some reason...  
In `Cleanup()`, I think you can just call `TestUtilities.ClearVisualTreeRoot()` instead.